dotnet/targets/Xamarin.Shared.Sdk.targets

```diff
@@ -507,6 +510,26 @@
 	</Target>
 
 	<!-- App bundle creation tasks -->
+
+	<Target Name="_CreateRuntimeConfiguration"
+		Condition="'$(UseMonoRuntime)' != 'false'"
```

Comment:
You might also check here if `$(GenerateRuntimeConfigurationFiles)` is `true`?

If a user sets it to `false` in their project, I think the `*.runtimeconfig.json` file would not be created:

https://github.com/dotnet/sdk/blob/415f1b90e14ddbbcfd6e2cb2d91b27e72db54db0/src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.targets#L238

Reply:
Hm yeah, good point, maybe we could even warn that some features won't work in that case.